TDDProject/Program.cs

```diff
@@ -6,6 +6,7 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
```

Comment:
I am approving this , but we do add test cases for swagger too as part of TDD if we use this in program.cs

Reply:
noted